common/components/urls/urls_v2.json

```diff
@@ -16,6 +16,7 @@
   {"name": "FindEvents", "pattern": "events"},
   {"name": "Press", "pattern": "about/news"},
   {"name": "PartnerWithUs", "pattern": "about/partner", "deprecated": true},
+  {"name": "Privacy", "pattern": "about/privacy"},
```

Comment:
This can be a regular expression, and combined with constants.py, handles the Django side of page routing.